Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialFunctions/EnhancedParallaxDepth.azsli

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Features/ParallaxMapping.azsli>
+#include "../MaterialInputs/ParallaxInput.azsli"
+#include <Atom/Features/MatrixUtility.azsli>
+
+ void EnhancedSetPixelDepth(
```

Comment:
Unclear why this needed to be factored out if only used once.

Reply:
https://github.com/o3de/o3de/pull/7437#discussion_r801076507